### DISCUSSION TITLE
NPCs in this mod could use a uniform script name and creature file with EET

### DISCUSSION THREAD

**Discussion by MephistoSatanDevil** [Spellhold-Studios/BG1-NPCs-for-SoA-ToB#6] [Suggestions]
When this mod is installed on EET, the npc script name within the mod is different from the in-game npc script name, and if when these characters have joined a party before in the game and join the game again in the SoA section, the characters' attributes and equipment may be different from the ones that have been adjusted by the player who has made additions in the previous game. Therefore, it is suggested that the NPCs in this mod use a uniform script name and creature file to make the gameplay experience more consistent.

**Comment by Gitjas** [comment]
It's a nice idea, but a loooot of work.

**Comment by MephistoSatanDevil** [comment]
I tried my best. Is there anyone else who can help complete this task?
https://github.com/MephistoSatanDevil/BG1_NPCs_for_EET

**Comment by Austin-BG** [comment]
Hello! I am not ready at the moment to make such large-scale changes, it would require reworking all the mod files and full testing (to avoid bugs). Maybe I'll do it sometime in the future, but I can't promise.
